app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_posts_counter
+    user.increment!(:PostsCounter)
```

Comment:
- [ ] This should be author.increment. Please update it

Reply:
- [ ] Please rename :PostCounter to posts_counter because capital letters are not a conventionally used naming scheme in Ruby on Rails.